packages/performance-tests/test/setup.ts

```diff
@@ -0,0 +1,8 @@
+import assert from "assert";
+import dotenv from "dotenv";
+
+const { error } = dotenv.config();
+
+if (error)
+  console.log("no env file found");
```

Comment:
```suggestion
  console.error("no env file found");
```

don't you also want to throw/exit after this?